### DISCUSSION TITLE
Install Elastic Search

### DISCUSSION THREAD

**Discussion by XenForo-FR** [leokhoa/laragon#394] [Q&A]
Hello,
I follow this guide to install Elastic Search : https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/installation.html
Now, how can i see the server where it is installed ? Which name server (localhost i guess) and which port ?
Thanks for your help.
Nicolas

**Comment by Pen-y-Fan** [comment]
What are you trying to do?
Why are you trying ElasticSearch?
Have you followed all the instructions to set up the client? 

The [connecting](https://www.elastic.co/guide/en/elasticsearch/client/php-api/current/connecting.html) step is to obtain the API keys and connect your local client to ElasticSearch using their API.

If you are testing search options you could try **[MeiliSearch](https://github.com/meilisearch/meilisearch-php)**, which will be available on http://127.0.0.1:7700 by default.

**Comment by XenForo-FR** [comment]
Hello, thanks for your answer.
I installed XenForo on a local server and i want to connect it to ElasticSearch, i have the add-on for that but i need to identify the ES server URL and port, probably 127.0.0.1 but idk the port.

![image](https://user-images.githubusercontent.com/124434978/216773415-ab6425b8-c03c-429d-a8ca-411e5b09d083.png)

I think i need to follow the process to setup the client, you're right.

**Comment by SarahTrees** [comment]
Elastic Search is a great thing!
But I think Laragon is the wrong contact.
Please go to the ElasticSearch forum or StackOverflow for support with your installation.
Please close this request, so Laragon can take care of the things it was made for. 
Have a good start with ElasticSearch!
Thank you and best regards.
